src/main/java/uk/ac/cam/cl/dtg/isaac/api/GameboardsFacade.java

```diff
@@ -89,26 +89,15 @@
 @Path("/")
 @Tag(name = "/gameboards")
 public class GameboardsFacade extends AbstractIsaacFacade {
+  private static final Logger log = LoggerFactory.getLogger(GameboardsFacade.class);
```

Comment:
Were these variables moved here because CheckStyle was complaining, or is this more of a personal preference (e.g. it looks tidier to have the `static` first, then `final`, then the rest)? If it's the former, then that's fair enough. If it was moved here because of personal preference, please move them back to exactly the same place where they were. There are multiple reasons for this: it makes it a lot harder to review (since it's hard to tell what's new, what was moved, and if the things that were moved have also been changed), it pollutes the git history (since it's not clear why these changes were done) and it's a personal preference, so what's stopping the next developer to reorder them differently?

I know it is painful to move everything back to where it was, but I hope the reasons I mentioned make sense. I also noticed that other methods in other files have also been moved, so they should be restored too.

Reply:
Those were changed due to Checkstyle so that I wouldn't get any Checkstyle errors similar to the one above on my PR despite not working on this part of the code and they were easier changes to make without causing any interference with previous code.

Reply:
Hmmm... This is very strange, because we have had CheckStyle for quite a while, and we've never had to reorder any functions or properties. I just ran CheckStyle locally on this file, and it does not complain about the order. Are you sure this isn't just a suggestion from your IDE?

Reply:
![Screenshot 2024-01-05 at 09 42 38](https://github.com/isaaccomputerscience/isaac-api/assets/82041490/2a8144ef-395e-41c7-9aa2-fddafec4ede7)
When I have switched it back to how it was originally, I get an error from Checkstyle.

Reply:
I think that may be a different configuration, because the rules say "active configuration". If you select the STEM rules, that error shouldn't appear.